lib/instrumentation/__snapshots__/index.spec.ts.snap

```diff
@@ -0,0 +1,137 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`instrumentation/index activate and remote logger 1`] = `
```

Comment:
do we need those snapshots?

Reply:
This is to ensure that the correct processors and exporters are enabled. I haven't managed to get it working with objects, so this is the fallback